cli/ballerina-cli/src/main/resources/cli-help/ballerina-shell.help

```diff
@@ -0,0 +1,33 @@
+NAME
+       ballerina-shell - Run ballerina interactive REPL
+
+SYNOPSIS
+       bal shell [-d|--debug] [-f|--force-dumb]  [-t|--time-out]
+
+
+DESCRIPTION
+       Shell runs a REPL instance of ballerina to enable users to run small
+       snippets to code to check their outputs. Please not that this is an
+       experimental feature.
+
+       Debug messages can be enabled via -d parameter.
+
+OPTIONS
+       -d
+           Enable debug mode from the beginning. Not enabled by default.
```

Comment:
Do we need the " from the beginning" part? This cannot be enabled after, right?
```suggestion
           Enable debug mode. Disabled by default.
```

Reply:
Also I don't think we need to mention the default for debug and force-dumb. The options indicate that we're toggling it on.

Reply:
The user can enable by debug mode later using `/debug` command while the shell is running. I'll remove the default value mention.

Reply:
Oh okay. But then do we need this option at all? The user can do `/debug` at the very beginning? Is there anything different between the two?

Reply:
The user would miss the initialization debug messages (if there are any that is) if they toggled it later.

Reply:
Yeah, then we'll have to have both.